build/Publish.targets

```diff
@@ -53,7 +53,7 @@
 
     <ItemGroup>
       <ForPublishing>
-        <RelativeBlobPath>$(Product)/$(FullNugetVersion)/$([System.String]::Copy('%(Filename)%(Extension)').Replace('\' ,'/'))</RelativeBlobPath>
+        <RelativeBlobPath>$(BlobStoragePartialRelativePath)/$(FullNugetVersion)/$([System.String]::Copy('%(Filename)%(Extension)').Replace('\' ,'/'))</RelativeBlobPath>
```

Comment:
Are any similar changes needed in `PublishContent.targets`?

Reply:
Oh, I think I see--this file's `GatherItemsForPattern` overrides the `GatherItemsForPattern` in `PublishContent.targets`, so no changes are needed. I'm not used to looking for target overriding.